docs/contents/components/other/close-button/accessibility.ja.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: CloseButton
+description: "CloseButtonは、基本的にコンポーネントの閉じる機能をトリガーするために使用するコンポーネントです。"
+package-name: "@yamada-ui/close-button"
```

Comment:
It seems the `order` is missing: https://github.com/yamada-ui/yamada-ui/pull/2288/commits/44f096087fd3917d479f36f6c5475a270006b33c.

Considering those of other components, I guess the appropriate order will be `3` (`1` for Props and `2` for Theming).